### ISSUE TITLE
NoUnused.Exports flags exports using records

### ISSUE THREAD

**Issue by sporto** [jfmengels/elm-review-unused#9]
**Describe the bug**

When a module imports a record using `(...)`. NoUnused.Exports can flag it as unused in the other module.

**SSCCE (Short Self-Contained Correct Example)**

See https://github.com/sporto/elm-review-exports-repro

1. Create a module exposing a record e.g. `person`
2. In another module import that e.g. `import Other exposing (..)`.
3. Use `person` to make another record e.g.
```
otherPerson = { person | age = 10 }
```
4. See error when running elm-review

```
-- ELM-REVIEW ERROR ----------------------------------------- src/Other.elm:1:24

NoUnused.Exports: Exposed function or value `person` is never used outside this
module.

1| module Other exposing (person)
                          ^^^^^^

This exposed element is never used. You may want to remove it to keep your
project clean, and maybe detect some unused code in your project.
```

**Expected behavior**
This export should not be flagged

**Additional context**
Importing explicilty removes this error e.g. `import Other exposing (person)`

**Comment by jfmengels** [comment]
Awesome, thank you! I'll try and take a look at that today.

**Comment by jfmengels** [comment]
Thanks for the report! The SSCCE was really useful, especially the part about the `exposing (..)` :rocket: 

I published the fix in v1.0.1 :slightly_smiling_face:

**Comment by sporto** [comment]
Thanks!
